forem/forem
# Add image composite element to feed if user exists

## What type of PR is this? (check all applicable)


## Description
Renders an `<image>` tag in the RSS feed. If the feed is a tag or the latest feed, user won't exist and we will omit this field.

## Related Tickets & Documents
Closes https://github.com/forem/forem/issues/9934

## QA Instructions, Screenshots, Recordings
<img width="779" alt="User feed" src="https://user-images.githubusercontent.com/3861088/95542243-1e289b00-09aa-11eb-8616-673cf4bb2677.png">

<img width="580" alt="Tag feed" src="https://user-images.githubusercontent.com/3861088/95542247-1ff25e80-09aa-11eb-88a9-6ea4ba9dd9eb.png">

<img width="562" alt="All Feed" src="https://user-images.githubusercontent.com/3861088/95542248-208af500-09aa-11eb-93e1-fe83f162b2a4.png">

**To test**:
- Login as a user with no organization.
- Navigate to `localhost:3000/feed`
  - No <image> tag should exist.
- Navigate to `localhost:3000/rss`
  - No <image> tag should exist.
- Navigate to `localhost:3000/feed/tag/<tagname>`
  - No <image> tag should exist.
- Navigate to `localhost:3000/feed/<username>`.
  - <image> tag should exist. Made up of <url>, <title>, <link> elems.

## Added tests?


## Added to documentation?


## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
